run_stats.h

```diff
@@ -28,11 +28,14 @@
 #include "memtier_benchmark.h"
 #include "run_stats_types.h"
 #include "JSON_handler.h"
+#include "deps/hdr_histogram/hdr_histogram.h"
+#include "deps/hdr_histogram/hdr_histogram_log.h"
 
-
-typedef std::map<float, int> latency_map;
-typedef std::map<float, int>::iterator latency_map_itr;
-typedef std::map<float, int>::const_iterator latency_map_itr_const;
+#define LATENCY_HDR_MIN_VALUE 1
```

Comment:
if the only usage of those #define's are in run_stats.h pls consider to move them (to run_stats.cpp)..